Orange/widgets/data/owcontinuize.py

```diff
@@ -242,7 +242,7 @@ def needs_dist(var, mtreat, ctreat):
                 for var, needs_dist in zip(domain.attributes, attr_needs_dist)]
     newclass = [continuize_var(var,
                                next(dist_iter) if needs_dist else None,
-                               class_treatment, Continuize.Remove)
```

Comment:
I can't find a reason for this Continuize.Remove?

Reply:
It doesn't hurt to give the value explicitly, either...